docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
You can remove all the `Makefile` and `.bat` files. They are just helpers, but better to just call `sphinx-build` directly (through pixi or tox).